tdi_python/tdiTable.py

```diff
@@ -1832,6 +1832,9 @@ def set_supported_apis_to_supported_commands(self):
                 if "dump" not in self.supported_commands:
                     self.supported_commands.append("dump")
 
+        if "add" not in self.supported_commands:
+            self.supported_commands.remove("add_from_json")
+
```

Comment:
Can we do it during initialization stage to at line 99
         self.supported_commands = ["info", "add_from_json", "entry", "string_choices"] and remove it from initialization for add_from_json?
if add then we can change this line to (self.supported_commands.append("add_from_json").

Reply:
done